### ISSUE TITLE
Usage with snowpack: '../web_modules/react-konva.js' does not provide an export named 'Rect'

### ISSUE THREAD

**Issue by ludwignagelhus** [konvajs/react-konva#542]
I am testing out the Snowpack bundler these days.
Previously I have used react-konva in create-react-app projects without any problems.

When I try to use react-konva in a snowpack project I get the following error:

```
Unhandled Runtime Error
Uncaught SyntaxError: The requested module '../web_modules/react-konva.js' does not provide an export named 'Rect'
```

I use another library with snowpack (react bootstrap) without problems.

I wrote the following component to try and find whats causing the trouble:

```
const ReactKonvaTest = () => {
  return (
    <>
      <Stage width={800} height={500}>
        <Layer>
          <Rect width={200} height={100} strokeWidth={10} stroke="#222" />
        </Layer>
      </Stage>
    </>
  );
};
```

...but I haven't been able to find a solution.

**Comment by ludwignagelhus** [comment]
The problem seems to be with snowpack:
https://www.snowpack.dev/reference/common-error-details#uncaught-syntaxerror%3A-the-requested-module-%E2%80%98%2Fweb_modules%2Fxxxxxx.js%E2%80%99-does-not-provide-an-export-named-%E2%80%98yyyyyy%E2%80%99

**Comment by ludwignagelhus** [comment]
I created a dicussion on snowpacks repo regarding this issue. You may find the answer I got helpful:
https://github.com/snowpackjs/snowpack/discussions/2066

**Comment by lavrton** [comment]
I just published `react-konva@17.0.1-0`. Probably it will fix the issue. Please try it.

**Comment by ludwignagelhus** [comment]
Thank you, but I am still getting the same error.

**Comment by lavrton** [comment]
Fixed with `react-konva@17.0.1-1`.
